exec/java-exec/src/test/java/org/apache/drill/TestBugFixes.java

```diff
@@ -24,8 +24,10 @@
 import java.util.List;
```

Comment:
I wonder if the tests should move to a file that is a bit more descriptive? Do we have tests for string functions? Else, future folks won't know that the code came from a "bug fix" and to look in this file.

Also, should we have tests for all the functions that were fixed? Presumably those tests don't exist because otherwise we would have found the bugs sooner.